app/screens/home/channel_list/servers/servers_list/server_item/server_item.tsx

```diff
@@ -158,17 +158,9 @@ const ServerItem = ({highlight, isActive, server, tutorialWatched}: Props) => {
     };
 
     const removeServer = async () => {
-        if (server.lastActiveAt > 0) {
-            await logout(server.url);
-        }
-
-        if (isActive) {
-            dismissBottomSheet();
-        } else {
-            DeviceEventEmitter.emit(Events.SWIPEABLE, '');
-        }
-
-        await DatabaseManager.destroyServerDatabase(server.url);
+        const skipLogoutFromServer = server.lastActiveAt === 0;
+        await dismissBottomSheet();
```

Comment:
In this case, we don't have to pass the Screen name to dismissBottomSheet?

Reply:
the default param is `Screens.BOTTOM_SHEET`, which in this case is the screen being displayed.